specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/Servers.json

```diff
@@ -931,14 +931,15 @@
           "type": "string"
         },
         "publicNetworkAccess": {
-          "description": "Whether or not public endpoint access is allowed for this server.  Value is optional but if passed in, must be 'Enabled' or 'Disabled'",
+          "description": "Whether or not public endpoint access is allowed for this server.  Value is optional but if passed in, must be 'Enabled' or 'Disabled' or 'SecuredByPerimeter'",
           "enum": [
             "Enabled",
-            "Disabled"
+            "Disabled",
+            "SecuredByPerimeter"
```

Comment:
>SecuredByPerimeter [](http://example.com/codeflow?start=13&length=18)

Interesting. What does this value mean?

Reply:
I never touched any of this. Not sure why swagger generator is picking up the changes.